core/hex.py

```diff
@@ -48,6 +50,16 @@ def neighbors(self):
             HEX_NEIGHBORS[self_tuple] = [self.add(d.value) for d in HexDirection]
         return HEX_NEIGHBORS[self_tuple]
 
+    def neighbors_se(self):
+        """
+        Finds the SW/SE/E neighbors of this cell.
+        :return: a list[Hex]
+        """
+        self_tuple = (self.x, self.y)
+        if self_tuple not in HEX_NEIGHBORS_SE:
+            HEX_NEIGHBORS_SE[self_tuple] = [self.add(d.value) for d in NEIGHBORS_SE]
+        return HEX_NEIGHBORS_SE[self_tuple]
```

Comment:
Used for `is_quiescent`. I didn't like the idea of defining something in core to accommodate a single usage but it felt equally icky adding generic functionality to a specialized agent class? I'm not sure there's other uses for this than color adjacency. Now I'm second guessing myself lol